jdaviz/configs/specviz2d/helper.py

```diff
@@ -152,6 +154,7 @@ def load_data(self, spectrum_2d=None, spectrum_1d=None, spectrum_1d_label=None,
                 timeout=timeout
             )
 
+    @deprecated(since="4.2")
```

Comment:
No alternative?

Reply:
ah, I had meant to either move this logic or postpone it.  I'll revisit if its feasible to move to the plugin here or better as a follow-up effort (in which case I'll remove the deprecation for now)

Reply:
I think ultimately this needs to be absorbed into the (future) general `load_data`, so I'll make a note on that ticket and hold off on deprecating until that's in place.